docs/release-notes/8.10.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[release-notes-header-8.10.0]]
+== 8.10
+
+[discrete]
+[[release-notes-8.10.0]]
+=== 8.10.0
+
+[discrete]
+[[breaking-changes-8.10.0]]
+==== Breaking changes
+There are no breaking changes in 8.10.0.
+
+[discrete]
+[[features-8.10.0]]
+==== New features
+* Adds a component under the Elastic AI Assistant header that allows you to select a different connector ({pull}163666[#163666]).
+* Adds a flyout that allows you to examine rule details when installing or updating a prebuilt rule ({pull}163304[#163304]).
+* Adds the ability to specify custom highlighted fields for an alert ({pull}163235[#163235]).
+* Adds role-based access control privileges for the Elastic AI Assistant ({pull}163031[#163031]).
+* Adds the **Reputation service** option to the malicious behavior protection setting on the Elastic Defend integration policy ({pull}161617[#161617]).
```

Comment:
We're missing a mention of the MITRE ATT&CK coverage page. It was implemented in the following PRs which were incorrectly labeled as `release_note:skip`:

- https://github.com/elastic/kibana/pull/161556
- https://github.com/elastic/kibana/pull/163498
- https://github.com/elastic/kibana/pull/164613
- https://github.com/elastic/kibana/pull/164986

Fixed all 4 to be `release_note:feature`.

Reply:
Thanks, @banderror! What should the summary for the new MITRE ATT&CK coverage page be?

Reply:
Thanks for catching this and updating the PR labels, @banderror! I added a summary for this PR in [7b5459e](https://github.com/elastic/security-docs/pull/3781/commits/7b5459ef15a2156cb75fe23ada95d5cd2e50f255).